front-end-side/src/Pages/ExpensesPages.js

```diff
@@ -47,6 +81,13 @@ const ExpensesPages = ({ categoryType }) => {
                                                 <td>{seller}</td>
                                                 <td>{purpose}</td>
                                                 <td>{parseFloat(amount).toFixed(2)}</td>
+                                                <td>
```

Comment:
Isn't it the case that food would be shown even thought if we are in food category?

Reply:
good point, will fix it